go/roothash/api/block/message.go

```diff
@@ -1,6 +1,43 @@
 package block
 
-// Message is a roothash message that can be sent by a runtime.
+import (
+	"fmt"
+
+	"github.com/oasisprotocol/oasis-core/go/common/crypto/hash"
+)
+
+// Message is a message that can be sent by a runtime.
 type Message struct {
-	// No valid messages are currently defined.
+	Noop *NoopMessage `json:"noop,omitempty"`
```

Comment:
The Noop message here just to ensure that runtime message propagation/processing works, while we don't have any other messages? Maybe add a comment explaining the need/usage of the Noop message.

Reply:
Yeah, I could remove these in #3511 even.